easy-random-core/src/test/java/org/jeasy/random/beans/ChainedSetterBean.java

```diff
@@ -23,16 +23,27 @@
  */
 package org.jeasy.random.beans;
 
-public class FluentSetterBean {
+public class ChainedSetterBean {
```

Comment:
The reason why this class is renamed.

"Fluent" word was used because of `commons-beanutils` `FluentPropertyBeanIntrospector`. Now it's not related.

The lombok project suggests these definitions:
**Fluent accessors**:
```java
class Data {

    private String value;

    public String value() {
        return value;
    }

    public Data value(String value) {
        this.value = value;
        return value;
    }
}
```

**Chained accessors**:
```java
class Data {

    private String value;

    public String getValue() {
        return value;
    }

    public Data setValue(String value) {
        this.value = value;
        return value;
    }
}
```